src/pages/AppList.svelte

```diff
@@ -84,8 +84,13 @@
             class="mzp-c-card-block-link"
             href="/apps/{app.app_name}"
             id="media-block"
+            data-glean-label="Home page: {app.app_name}"
           >
-            <div class="mzp-c-card-media-wrapper" id="media-wrapper">
+            <div
+              class="mzp-c-card-media-wrapper"
+              id="media-wrapper"
+              style="pointer-events:none"
```

Comment:
Please add a comment here explaining why we need this and reference the bug in which this might be fixed.

Reply:
Done.